doc/TROUBLESHOOTING.md

```diff
@@ -39,15 +43,16 @@ Add this snipet before you instantiate the player `Clappr.Log.setLevel(0)`
 
 Very often people open issues related to: **stream not working, freezing, glitching, stopping and so on.** You can try the steps bellow, taking notes about the results:
 
-* try to run the same example at [CDN](http://cdn.clappr.io)
```

Comment:
Is there any reason for this change? There seems to be no need to change the bullet style in this documentation as the final result is the same.